go-controller/pkg/ovn/egressfirewall_test.go

```diff
@@ -615,13 +620,14 @@ var _ = ginkgo.Describe("OVN EgressFirewall Operations", func() {
 					_, err = fakeOVN.fakeClient.KubeClient.CoreV1().Nodes().Patch(context.TODO(), nodeName,
 						types.MergePatchType, patchData, metav1.PatchOptions{})
 					gomega.Expect(err).NotTo(gomega.HaveOccurred())
-					asHash, _ := getNsAddrSetHashNames(namespace1.Name)
+					as4Hash, as6Hash := getNsAddrSetHashNames(namespace1.Name)
 					dbIDs := fakeOVN.controller.getEgressFirewallACLDbIDs(egressFirewall.Namespace, 0)
-					ipv4ACL := libovsdbops.BuildACL(
+					ipv46ACL := libovsdbops.BuildACL(
 						libovsdbutil.GetACLName(dbIDs),
 						nbdb.ACLDirectionToLport,
 						t.EgressFirewallStartPriority,
-						fmt.Sprintf("(ip4.dst == %s) && ip4.src == $%s", nodeIP, asHash),
+						fmt.Sprintf("(ip4.dst == %s || ip4.dst == %s || ip6.dst == %s) && (ip4.src == $%s || ip6.src == $%s)",
```

Comment:
lol feels weird having v4 and v6 in the same match, the `||` is fine, but its combined by an `&&` ?